R/subject_industry.R

```diff
@@ -905,12 +919,12 @@ crosstab_text <- function(tables_data_grouped, subjectinput, YAGinput, countinpu
 
         regiontext <- paste(
           "<b>",
-          uniqueregions[1], "</b> was the most common industry for those currently living in ", textprod(data1), ",
-                      <b>", uniqueregions[2], "</b> was the most common industry for those living in ", textprod(data2), ", <b>",
-          uniqueregions[3], "</b> was the most common industry for those living in ", textprod(data3), ", <b>",
-          uniqueregions[4], "</b> was the most common industry for those living in ", textprod(data4), ", <b>",
-          uniqueregions[5], "</b> was the most common industry for those living in ", textprod(data5), " and <b>",
-          uniqueregions[6], "</b> was the most common industry for those living in ", textprod(data6), ".",
+          uniqueregions[1], " for those currently living in ", textprod(data1), ",
+                      <b>", uniqueregions[2], " for those living in ", textprod(data2), ", <b>",
+          uniqueregions[3], " for those living in ", textprod(data3), ", <b>",
+          uniqueregions[4], " for those living in ", textprod(data4), ", <b>",
+          uniqueregions[5], " for those living in ", textprod(data5), " and <b>",
+          uniqueregions[6], " for those living in ", textprod(data6), ".",
           sep = ""
         )
       } else {
```

Comment:
Just trying to understand this textarray section below... you have if statements up to length(uniqueregions)=6, but there nine regions, so would any combination above six (if there even is any) be captured by the textarray code? Is your comment on the textarray section saying it would work for all the uniqueregions bit (i.e. could replace all the if statements?) but you've just left it in the final else area just in case it wouldn't work?

Reply:
Yes I believe that's right - that @rmbielby added an 'else' statement to encapsulate everything but just didn't delete the previous code.